### PR TITLE
Added Segment<>Novu integration guide [Ticket DVRL-57]

### PR DESCRIPTION
**Pull request by iampearceman**
https://github.com/novuhq/docs/assets/63902456/b77163ae-ffb6-4973-814f-72d527269079

### CODE REVIEW

File: guides/how-to-integrate-segment-with-novu.mdx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+### Sending a payload to Novu
+
+Now let's learn how to send a payload to Novu based on the `onTrack` event function:
+
+To successfully send a payload to Novu, triggering the appropriate workflow and delivering it to the correct customer, please ensure that you provide the following information:
+
+1. Novu's Workflow Identifier.
+2. Subscriber ID (Please ensure that the Subscriber ID is both unique and consistent across all of your databases).
+3. Your Novu API Key.
+4. Novu's API Endpoint: [https://api.novu.co/v1/events/trigger]
```
Comment (unicodeveloper) [inline guides/how-to-integrate-segment-with-novu.mdx:143]: I am not sure this part is linked properly.

File: guides/how-to-integrate-segment-with-novu.mdx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+For example, if you rename this function `onIdentify`, it listens for “Identify” events instead.
+
+### Sending a payload to Novu
+
+Now let's learn how to send a payload to Novu based on the `onTrack` event function:
+
+To successfully send a payload to Novu, triggering the appropriate workflow and delivering it to the correct customer, please ensure that you provide the following information:
+
+1. Novu's Workflow Identifier.
```
Comment (unicodeveloper) [inline guides/how-to-integrate-segment-with-novu.mdx:140]: Can you link this to the workflow section or the workflow concept section?

**Review by unicodeveloper — COMMENTED** [review]
So, I noticed we don't have a sample app for this guide. We need one so that people can just clone and also run it @iampearceman

File: guides/how-to-integrate-segment-with-novu.mdx (hunk 1 above)
Comment (iampearceman) [reply guides/how-to-integrate-segment-with-novu.mdx]: @unicodeveloper What do you mean?

File: guides/how-to-integrate-segment-with-novu.mdx (hunk 2 above)
Comment (iampearceman) [reply guides/how-to-integrate-segment-with-novu.mdx]: Done.

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**